src/js/components/contact.js

```diff
@@ -0,0 +1,38 @@
+import { newText, newContainer } from './helpers';
+
+/**
+ * @param {Array [objectInformation]} informationArray esta función recibe un arreglo de objetos
+ * y retorna una lista de elementos de información desde la data acerca del contacto con la fundación.
+ * Toma el array de 'informationArray' le hace un forEach que nos crea un elemento por cada una de las
+ * veces que encuentre información en la data e ingresa esta información en un contenedor,
+ * así mismo crea una clase del contenedor y posterior imprime la información encontrada según el orden de
+ * la data.
+ */
+const nodeInformation = (informationArray) => {
+  const containerInformation = newContainer('div', [], ['contactUs__container-Information']);
+  informationArray.forEach((item) => {
+    const title = newText('h3', item.label, ['contactUs__label']);
+    const paragraph = newText('p', item.texts, ['contactUs__text']);
```

Comment:
Here, you need to go through the array `item.texts` an render each text in a separate line.